README.md

```diff
@@ -23,6 +23,19 @@ work in progress.
 Please file reports on the
 [GitHub Issue Tracker](https://github.com/dart-lang/dart-pad/issues).
 
+## Running locally
+
+The project contains a tiny appengine for redirecting to gists and the mobile UI. To run this:
+- Download the AppEngine SDK for Python here: https://cloud.google.com/appengine/downloads
+- Navigate to the root of the checkout folder
+- run `pub get`
```

Comment:
It doesn't seem to do the right thing if I do. I ran into this earlier this evening. Lets debug next week.